.github/workflows/linux.yml

```diff
@@ -13,7 +13,7 @@ concurrency:
 
 jobs:
   linux:
-    name: ${{ matrix.distro }}
+    name: ${{ matrix.distro }}${{ matrix.platform != 'linux/amd64' && format(' ({0})', matrix.platform) || '' }}
```

Comment:
Redundant space before `({0...)`.

Reply:
```suggestion
    name: ${{ matrix.distro }}${{ matrix.platform != 'linux/amd64' && format('({0})', matrix.platform) || '' }}
```

This space? It's been added intentionally.

Reply:
Ah! I didn't see the final github job names!